config/webpack/postcss/ThemePlugin.js

```diff
@@ -0,0 +1,61 @@
+const fs = require('fs-extra');
+const path = require('path');
+const postcss = require('postcss');
+const removeCssModulesPseudoClasses = require('./_removeCssModulesPseudoClasses');
+
+const CONFIG = 'terra-theme.config.js';
+const SUPPORTED_THEMES = [
+  'orion-fusion-theme',
+  'clinical-lowlight-theme',
+  'cerner-clinical-theme',
+];
+
+/**
+ * The purpose of this plugin is to create a default theme from a scoped theme
+ * and to remove any supported themes that are not desired.
+ */
+module.exports = postcss.plugin('postcss-test-plugin', (config) => {
```

Comment:
postcss-test-plugin?  is that a convention or is there a better name here?

Reply:
there is definitely a better name, that was copied from the guide i started with on writing post css plugins :)